recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
     number: 0
-    skip: True  # [py35]
+    skip: True  # [py3k]
```

Comment:
Right. So my recommendation would be to change this to `py>34` instead. This has the same effect, but it is clearer in its meaning. Plus, if people want a working Python 3.4 environment, they can still build one locally.